packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dedicatedCloud.service.js

```diff
@@ -583,6 +583,21 @@ class DedicatedCloudService {
     });
     return filterRequest;
   }
+
+  postGrantActiveDirectory(serviceName, activeDirectoryId, params, type) {
+    const userType = 'user';
+
+    return this.$http
+      .post(
+        `/dedicatedCloud/${serviceName}/federation/activeDirectory/${activeDirectoryId}/${
+          type === userType
+            ? 'grantActiveDirectoryUser'
+            : 'grantActiveDirectoryGroup'
```

Comment:
```suggestion
          `grantActiveDirectory${type}`
```
or 
```suggestion
          type === 'user'
            ? 'grantActiveDirectoryUser'
            : 'grantActiveDirectoryGroup'
```
no need to declare a constant variable for this imo

Reply:
It's applied